MovieTime/MovieTime/Core/Extensions/UIImageViewExtensions.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  UIImageViewExtensions.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 10/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+extension UIImageView {
+
+    func loadBackdropImage(filePath: String?, placeholderImage: UIImage? = nil, loadingView: MTLoadingProtocol? = nil, completion: MTImageRequestResponse = nil) {
+        guard let filePath = filePath, let service = MTApplicationCoordinator.shared.imageService else {
```

Comment:
clearly the filePath should not be optional, because if I pass nil, I'll receive back an error. When you say in the signature that something is optional, you're saying it makes sense to be nil, but in this case I'd say it doesn't make sense

there are some cases where we would have to add guards all over the project because we call a method so many times, so I start seeing more sense on moving the guard inside the method and doing something sensible with it. It might be your case here. I just wanted to bring this as a point of attention :)